Gruntfile.js

```diff
@@ -236,9 +236,6 @@ module.exports = function (grunt) {
             },
             test: {
                 src: ['dist/*.js'],
-            },
```

Comment:
I've dropped the testDebug target, now that the spec runner file is ignore, I don't see any harm in keeping this file hanging around. I find it useful for debugging.